.gitignore

```diff
@@ -99,11 +93,11 @@ tags
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
 
 # Build artifacts
-build
+/build
```

Comment:
so you only want to match `build` directories but allow files named `build`. :+1:

Reply:
IIUC, this is specifically the build directory at the root of the project, all other files and directories named `build` will not be ignored.